documentation/modules/rn-2.6.adoc

```diff
@@ -0,0 +1,94 @@
+// Module included in the following assemblies:
+//
+// * documentation/doc-Release_notes/master.adoc
+
+[id="rn-26_{context}"]
+= {project-full} 2.6
+
+Migrate virtual machines (VMs) from VMware vSphere or {rhv-full} or {osp} to {virt} with {the-lc} {project-first}.
+
+The release notes describe technical changes, new features and enhancements, and known issues.
+
+[id="technical-changes-26_{context}"]
+== Technical changes
+
+// {project-short} {project-z-version} is a maintenance release. There are no technical changes.
+
+This release has the following technical changes:
+
+.Migration from OpenStack moves to being a fully supported feature
+
+In this version of {project-short}, migration using OpenStack source providers graduated from a Technology Preview feature to a fully supported feature.
+
+.Disabling FIPS
+
+EMS enforcement is disabled for migrations with VMware vSphere source providers to enable migrations from versions of vSphere that are supported by {project-short} but do not comply with the 2023 FIPS requirements.
+
+.Integration of the create and update provider user interface
+
+The user interface of the create and update providers now aligns with the look and feel of the {ocp} web console and displays up-to-date data.
```

Comment:
```suggestion
The user interface of the create and update providers now aligns with the look and feel of the {ocp} web console and displays up-to-date data.
```

@RichardHoch - please can I check how are the providers called in the UI?

Reply:
@anarnold97 @HagayVider1 They are called source providers and destination providers.

Reply:
corrected

Reply:
You have removed this correction. Why?